edgelet/edgelet-image-cleanup/src/error.rs

```diff
@@ -0,0 +1,17 @@
+#[derive(Debug, thiserror::Error)]
+pub enum ImageCleanupError {
+    #[error("image id was unexpectedly found to be empty")]
+    GetImageId(),
```

Comment:
```suggestion
    GetImageId,
```

Reply:
This doesn't work as-is. Inclined to ignore since it's a non-functional change.